script.js

```diff
@@ -0,0 +1,151 @@
+const form = document.querySelector('form')
+const splash = document.querySelector('.splash')
+const gameBoard = document.querySelector('.game-board')
+
+/** Generates the content for the grid container, with default miss class and unique data-grid-id attributes
+ *
+ * @param height inputted height
+ * @param width inputted width
+ * @returns {string} innerHTML for the grid container
+ */
+ function gridContent(height, width){
+    let innerHTML = ''
+    let gridSize = height * width
+    for (let i = 1; i <= gridSize; i++){
+        let text = `<div class="game-item game-board__sapling miss" data-grid-id="${i}"></div>`
+        innerHTML += text
+
+    }
+    return innerHTML
+}
+
+/** Sets the grid template for the container
+ *
+ * @param height inputted height
+ * @param width inputted width
+ */
+function gridDefinition(height, width){
+    let gridCont = document.querySelector(".game-board__grid")
+    gridCont.innerHTML= gridContent(height, width)
+    gridCont.style.gridTemplateColumns = `repeat(${width}, 1fr)`
+    gridCont.style.gridTemplateRows = `repeat(${height}, 1fr)`
+}
+
+/**
+ * This function changes game-board tiles on click based on hit or miss class
+ *
+ * @param e
+ */
+function hitOrMiss(e) {
+    let target = e.currentTarget.classList
+    if (target.contains('game-board__carrot') ||target.contains('game-board__gopher')) {
+        return
```

Comment:
You can then remove this - since no code will run unless the target isn't a gopher/carrot